diff --git a/utils/build/build.js b/utils/build/build.js
--- a/utils/build/build.js
+++ b/utils/build/build.js
@@ -237,37 +237,6 @@ function copyFile(file, from, to) {
   fs.copyFileSync(file, destination);
 }
 
-class GroupStep extends Step {
-  /** @param {Step[]} steps */
-  constructor(steps) {
-    super({ concurrent: false });
-    this._steps = steps;
-    if (steps.some(s => !s.concurrent))
-      throw new Error('Composite step cannot contain non-concurrent steps');
-  }
-  async run() {
-    console.log('==== Starting parallel group');
-    const start = Date.now();
-    await Promise.all(this._steps.map(step => step.run()));
-    console.log('==== Parallel group finished in', Date.now() - start, 'ms');
-  }
-}
-
-/** @type {Step[]} */
-const updateSteps = [];
-
-// Update test runner.
-updateSteps.push(new ProgramStep({
-  command: 'npm',
-  args: ['ci', '--save=false', '--fund=false', '--audit=false'],
-  shell: true,
-  cwd: path.join(__dirname, '..', '..', 'tests', 'playwright-test', 'stable-test-runner'),
-  concurrent: true,
-}));
-
-
-steps.push(new GroupStep(updateSteps));
-
 // Build injected icons.
 steps.push(new ProgramStep({
   command: 'node',
@@ -868,7 +837,7 @@ function copyXdgOpen() {
   console.log('==== Copied xdg-open to', path.join(outdir, 'xdg-open'));
 }
 
-// Copy xdg-open after bundles 'npm ci' has finished.
+// Copy xdg-open after generated bundles are available.
 steps.push(new CustomCallbackStep(copyXdgOpen));
 
 function pkgNameFromPath(p) {
