--- lopdf-0.44.0/src/datetime.rs
+++ lopdf-0.44.0/src/datetime.rs
@@ -103 +103 @@
-                    date.format(&FormatItem::StringLiteral("%Y%m%d%H%M%SZ")).unwrap()
+                    date.format(&FormatItem::Literal(b"%Y%m%d%H%M%SZ")).unwrap()
@@ -114 +114 @@
-                let format = time::format_description::parse_borrowed::<3>(
+                let format = time::format_description::parse_borrowed::<2>(
@@ -131 +131 @@
-            let format = time::format_description::parse_borrowed::<3>(
+            let format = time::format_description::parse_borrowed::<2>(
