--- comrak-0.54.0/Cargo.toml
+++ comrak-0.54.0/Cargo.toml
@@ -59,10 +59,8 @@
 
 [features]
 attributes = []
-bon = ["dep:bon"]
 cli = [
     "clap",
-    "bon",
     "shell-words",
     "xdg",
     "fmt2io",
@@ -71,9 +69,7 @@
     "attributes",
 ]
 default = [
-    "cli",
     "syntect-onig",
-    "bon",
 ]
 phoenix_heex = []
 shortcodes = ["emojis"]
@@ -157,10 +153,6 @@
 features = ["derive"]
 optional = true
 
-[dependencies.bon]
-version = "3"
-optional = true
-
 [dependencies.caseless]
 version = "0.2"

--- comrak-0.54.0/build.rs
+++ comrak-0.54.0/build.rs
@@ -3,6 +3,8 @@
 use std::{env, path::PathBuf};
 
 fn main() {
+    println!("cargo::rustc-check-cfg=cfg(feature, values(\"bon\"))");
+
     let out_dir: PathBuf = env::var("OUT_DIR").unwrap().parse().unwrap();
 
     // entity::lookup is handed just the inner entity name, like "amp" for
