--- bit-vec-0.10.1/Cargo.toml
+++ bit-vec-0.10.1/Cargo.toml
@@ -35,56 +35,16 @@
 license = "Apache-2.0 OR MIT"
 repository = "https://github.com/contain-rs/bit-vec"
 
-[package.metadata.docs.rs]
-features = [
-    "borsh",
-    "serde",
-    "miniserde",
-]
-
 [features]
-allocator_api = []
 default = ["std"]
-std = [
-    "serde?/std",
-    "borsh?/std",
-]
+std = []
 
 [lib]
 name = "bit_vec"
 path = "src/lib.rs"
 
-[[bench]]
-name = "bench"
-path = "benches/bench.rs"
-
-[dependencies.borsh]
-version = "1.7.0"
-features = ["derive"]
-optional = true
-default-features = false
-
-[dependencies.miniserde]
-version = "0.1"
-optional = true
-
-[dependencies.serde]
-version = "1.0"
-features = [
-    "derive",
-    "alloc",
-]
-optional = true
-default-features = false
-
-[dev-dependencies.generic-tests]
-version = "0.1"
-
 [dev-dependencies.rand]
 version = "0.10"
 
 [dev-dependencies.rand_xorshift]
 version = "0.5"
-
-[dev-dependencies.serde_json]
-version = "1.0"
