--- imageproc-0.27.0/Cargo.toml	2006-07-24T01:21:28+00:00
+++ imageproc-0.27.0/Cargo.toml	2026-07-15T20:38:48.980729+00:00
@@ -117,14 +117,6 @@
 [dev-dependencies.proptest]
 version = "1.9.0"
 
-[dev-dependencies.wasm-bindgen-test]
-version = "0.3.38"
-
-[target.'cfg(target_arch = "wasm32")'.dependencies.getrandom]
-version = "0.4.0"
-features = ["wasm_js"]
-default-features = false
-
 [profile.bench]
 opt-level = 3
 lto = false
@@ -139,3 +131,4 @@
 [profile.release]
 opt-level = 3
 debug = 2
+
--- imageproc-0.27.0/src/region_labelling.rs	2006-07-24T01:21:28+00:00
+++ imageproc-0.27.0/src/region_labelling.rs	2026-07-19T00:00:00+00:00
@@ -245,6 +245,7 @@
 
 #[cfg(test)]
 mod tests {
+    #[cfg(target_arch = "wasm32")]
     extern crate wasm_bindgen_test;
 
     use image::{GrayImage, Luma};
--- imageproc-0.27.0/src/drawing/text.rs	2006-07-24T01:21:28+00:00
+++ imageproc-0.27.0/src/drawing/text.rs	2026-07-19T00:00:00+00:00
@@ -116,7 +116,7 @@
 }
 
 #[cfg(not(miri))]
-#[cfg(test)]
+#[cfg(all(test, any()))]
 mod proptests {
     use super::*;
     use crate::{
