--- fearless_simd-0.4.1/Cargo.toml.orig	1970-01-01T00:00:01+00:00
+++ fearless_simd-0.4.1/Cargo.toml	2025-05-06T20:35:38.476135+00:00
@@ -15,7 +15,7 @@
 name = "fearless_simd"
 version = "0.4.1"
 edition = "2021"
-license = "Apache-2.0 OR MIT"
+license = "MIT OR Apache-2.0"
 authors = ["Luca Barbato <lu_zero@gentoo.org>"]
 build = false
 autolib = false
@@ -65,12 +65,6 @@
 version = "1.0.137"
 features = ["derive"]
 
-[dev-dependencies.trybuild]
-version = "1.0.83"
-
-[dev-dependencies.wasm-bindgen-test]
-version = "0.3.0"
-
 [dev-dependencies.zune-jpeg]
 version = "0.4.14"
