--- a/setup.py
+++ b/setup.py
@@ -51,7 +51,7 @@ setup(
     include_package_data=True,
     version='1.5.0',
     description='Utilities from sensAI, the Python library for sensible AI',
-    install_requires=relaxed_requirements_from_file("requirements.txt"),
+    install_requires=["typing-extensions>=4.6"],
     dependency_links=[],
     setup_requires=["wheel"],
     author='Dominik Jain et al.',
